cpp/abconv/Converter.cc

```diff
@@ -22,15 +22,15 @@ ab::abconv::Converter::Converter(std::shared_ptr<HepMC3::Reader> reader,
 void ab::abconv::Converter::convert() {
     using namespace HepMC3;
 
-    long int events_processed = 0;
+    uint64_t events_processed = 0;
     // HepMC files open
 
     // Event loop
     while( !_reader->failed() ) {
         GenEvent evt;
         _reader->read_event(evt);
         if (_reader->failed()) {
-            printf("End of file reached. Events processed: %li Exit.\n", events_processed);
+            printf("End of file reached. Events processed: %lu Exit.\n", events_processed);
```

Comment:
```suggestion
            printf("End of file reached. Events processed: %" PRIu64 " Exit.\n", events_processed);
```
IIRC uint64_t is neither long nor long long.